test/unit/controllers/mainCtrl.spec.js

```diff
@@ -127,10 +147,142 @@ describe('app.config,js', function ()
             {
                 expect(mainCtrl.wallet.GBP).toEqual(0);
             });
+            it('should return wallet.AUD return 0', function ()
+            {
+                expect(mainCtrl.wallet.AUD).toEqual(0);
+            });
+            it('should return wallet.CAD return 0', function ()
+            {
+                expect(mainCtrl.wallet.CAD).toEqual(0);
+            });
+            it('should return wallet.HUF return 0', function ()
+            {
+                expect(mainCtrl.wallet.HUF).toEqual(0);
+            });
             it('should return wallet.XDR return 0', function ()
             {
                 expect(mainCtrl.wallet.XDR).toEqual(0);
             });
         });
+
+        describe('getCurrency', function ()
+        {
+            beforeEach(function ()
+            {
+                mainCtrl.getCurrency();
+            });
+            it('should be call ExchangeRateService.getCurrency', function ()
+            {
+                expect(CurrencyServiceMock.getCurrency).toHaveBeenCalled();
+            });
+            describe('always', function ()
+            {
+                beforeEach(function ()
+                {
+                    mainCtrl.amountForBay = 10;
+                    mainCtrl.amountForSell = 10;
+                    mainCtrl.getCurrency('GBP');
+                });
+                it('should return actually rates in buy', function ()
```

Comment:
`should return actual purchase price` - aktualna cena kupna (w przybliżeniu:) )

`should return actual sell price` - aktualna cena sprzedaży